src/pages/viewPost/ViewPost.js

```diff
@@ -61,12 +73,21 @@ export default function ViewPost() {
       <Typography variant="h2">Comments</Typography>
       {post &&
         post.comments.map((comment, index) => (
-          <CommentCard comment={comment} key={index} />
+          <CommentCard
+            getProfilePic={getProfilePic}
+            navigateToProfile={navigateToProfile}
+            comment={comment}
+            key={index}
+          />
         ))}
       {post && post.comments.length === 0 && (
         <Typography>No comments found</Typography>
       )}
-      <CommentBox onSave={onPostAComment} />
+      <CommentBox
+        getProfilePic={getProfilePic}
+        navigateToProfile={navigateToProfile}
+        onSave={onPostAComment}
+      />
     </Wrapper>
   );
 }
```

Comment:
Suggestion: please check create post text area, fix the width of the text area, it is moving in horizontal direction if more text added, by moving right sidebar out of the screen.

Reply:
Thanks for finding this, I have fixed now